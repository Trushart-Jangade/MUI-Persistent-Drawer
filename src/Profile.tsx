import * as React from "react";
export default class Profile extends React.Component {
    public render() {
        return (  
                    <div>
                        <h1>Profile Component</h1>
                    </div>
        );
  }
}
