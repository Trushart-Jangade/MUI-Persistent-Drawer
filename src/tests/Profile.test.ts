import * as React from 'react';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import Profile from '../Profile';
const ReactShallowRenderer = require('react-test-renderer/shallow');
test('should render Profile correctly', () => {
    const renderer = new ReactShallowRenderer();
    renderer.render(React.createElement(Profile));    
    expect(renderer.getRenderOutput()).toMatchSnapshot();
});
