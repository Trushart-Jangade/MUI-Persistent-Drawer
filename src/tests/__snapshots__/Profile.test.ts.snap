// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Profile correctly 1`] = `
<div>
  <h1>
    Profile Component
  </h1>
</div>
`;
